assets/blocks/course-outline/style.scss

```diff
@@ -54,68 +41,161 @@ $dark-gray: #1a1d20;
 }
 
 .wp-block-sensei-lms-course-outline-module {
-	border: solid 1px $dark-gray;
-	margin: 1.5em 0;
+	$block: '.wp-block-sensei-lms-course-outline-module';
+
+	&.is-style-default, &.is-style-minimal {
+
+		#{$block}__progress-indicator {
+			border-radius: 999px;
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			margin-right: 20px;
+			margin-left: 20px;
+
+			&__text {
+				font-weight: normal;
+				margin: 2px 15px;
+				font-size: 14px;
+				white-space: nowrap;
+				text-transform: uppercase;
+			}
+
+			&.completed {
+				background-color: #1FD25A;
+				color: #fff;
+			}
+		}
 
-	&__name {
-		padding: 15px 15px 15px 25px;
-		background-color: $dark-gray;
-		font-size: 24px;
-		font-weight: bold;
-		color: #FFF;
-		display: flex;
-		align-items: center;
+		#{$block}__name {
+			padding: 1rem;
+			font-weight: bold;
+			display: flex;
+			align-items: center;
+		}
+
+		h2 {
+			font-family: inherit;
+			font-size: inherit;
+			color: inherit;
+			line-height: inherit;
+			margin: 0;
+			flex: 1;
+		}
+
+		#{$block}__description {
+			padding: 1rem;
+		}
+
+		#{$block}__lessons-title {
+			padding: 5px 1rem;
+			font-size: 11px;
+			font-weight: 600;
+			color: inherit;
+			text-transform: uppercase;
+		}
 	}
 
-	&__description {
-		padding: 25px;
-		color: $dark-gray;
-		font-size: 14px;
+	&.is-style-default {
+
+		#{$block}__lessons-title {
+			background-color: #EEE;
+		}
+		#{$block}__name {
+			&:not(.has-background-color) {
+				background-color: $dark-gray;
+			}
+			&:not(.has-color) {
+				color: #FFF;
+			}
+		}
+
+		#{$block}__lessons-title {
+			background-color: #EEE;
+		}
+		#{$block}__progress-indicator {
+			background-color: #FFF;
+			color: $dark-gray;
+		}
 	}
 
-	&__lessons-title {
-		padding: 5px 25px;
-		background-color: #EEE;
-		font-size: 11px;
-		font-weight: 600;
-		color: $dark-gray;
-		text-transform: uppercase;
+	&.is-style-minimal {
+		#{$block}__progress-indicator:not(.completed) {
+			border: 1px solid currentColor;
+		}
+
+	}
+
+	&__name__minimal-border {
+		display: block;
+		border: none;
+		height: 4px;
+		.wp-block-sensei-lms-course-outline.is-style-default & {
+			margin-left: 4px;
+			margin-right: 4px;
+		}
+		background-color: $dark-gray;
 	}
 }
 
 .wp-block-sensei-lms-course-outline-lesson {
 	display: flex;
 	align-items: center;
-	padding: 20px 25px;
+	margin: 1px 0;
+	position: relative;
+	font-size: 1rem;
+
 	> a {
+		display: block;
+		flex: 1;
 		color: inherit;
+		font-family: inherit;
+		font-size: inherit;
+		line-height: inherit;
+		text-decoration: inherit;
+		border: none;
+		font-weight: normal;
+		padding: 20px 1rem;
+		&:hover {
+			color: inherit;
+		}
 	}
 
 	.entry-content & {
 		margin: 0;
 	}
-	font-size: 16px;
 
-	&::before {
+	&:before {
```

Comment:
Out of curiosity, did we changed `::` to `:` for some specific reason?

Reply:
TIL there is a pseudo-element syntax, thought it was a mistake :) Changed back in 31a41d8